releasenotes/notes/support_excitedstatessolver_bopes-2bbb5a282b1ff582.yaml

```
@@ -0,0 +1,21 @@
+features:
+  - |
+    The solver of BOPESSampler now accepts solvers of type
+    `Union[GroundStateSolver,ExcitedStatesSolver]` instead of only a `GroundStateSolver`. 
+    This generalizes to excited states the sampling 
+    of nuclear coordinates for Born Oppenheimer Potential Energy Surfaces.
+    Adds the `get_qubit_operators()` in
+    :class:`qiskit_nature.algorithms.ExcitedStatesEigensolver`.
+    This matches the format of
+    :class:`qiskit_nature.algorithms.GroundStatesEigensolver`.
+    BOPESSampler now also accepts auxiliary operators to pass to the solver.
+    Geometry-dependent observable can also be defined as auxiliaries.
+    See the `Sampling the potential energy surface <tutorials/05_Sampling_potential_energy_surfaces.ipynb>`_ tutorial
+    for a demonstration of how to use this calculation of excited state Born Oppenheimer potential energy surfaces.
+deprecations:
+  - |
+    The argument `gss` of the constructor of:class:`qiskit_nature.algorithms.BOPESSampler` was deprecated 
```

```suggestion
    The argument `gss` of the constructor of :class:`qiskit_nature.algorithms.BOPESSampler` was deprecated 
```
There was no space between of and the class ref so it all came out as text in the release note rather than being converted.

---

Typo fixed